source/documentation/retire-packages.html.md

```diff
@@ -0,0 +1,50 @@
+---
+author: ykarel
+title: Retiring a package
+---
+
+# Retiring a package from RDO
+
+1. toc
+{:toc}
+
+### How to remove an OpenStack package from RDO Trunk
+
+Package for a project can exist across different releases. So retiring it in RDO needs to go via stages,
+To remove a package, following steps are required:-
+
+Stage 1:- Project Source is retired/deleted upstream, but it's package still need to be maintained because it's required by other projects
```

Comment:
nitpick, it'd be good to use bold format for "Stage X".

Reply:
Done